tests/e2e/cucumber/features/integrations/searchProjectSpace.ocis.feature

```diff
@@ -0,0 +1,42 @@
+Feature: Search in the project space
+    
+    Scenario: Search in the project spaces
+        Given "Admin" creates following users
+            | id    |
+            | Alice |
+        And "Admin" assigns following roles to the users
+            | id    | role       |
+            | Alice | SpaceAdmin |
+        When "Alice" logs in
+        And "Alice" opens the "files" app
+        And "Alice" navigates to the projects space page
+        And "Alice" creates the following project spaces
+            | name | id     |
+            | team | team.1 |
+        And "Alice" navigates to the project space "team.1"
+        And "Alice" creates the following resources
+            | resource                  | type   |
+            | folder(WithSymbols:!;_+-) | folder |
```

Comment:
```suggestion
            | folder(WithSymbols:!;_+-&) | folder |
```
Can we also add & here so that we know if regression occurs for this https://github.com/owncloud/ocis/issues/4474

Reply:
np, done